### ISSUE TITLE
Sprint G

### ISSUE THREAD

**Issue by whyrusleeping** [ipfs/pm#8]
**Sprint Goals**:

This sprint is mainly about fixing lots of bugs/clunkiness. We've let some annoying issues creep in, and we need to fix them. IPFS must be -- above all -- robust, dependable, and useful.

**Sprint Participants**:

- @whyrusleeping @krl @cryptix @wking @dylanPowers @jbenet @tv42

**Sprint Deliverables**:

(trying out a different format this week)

#### @wking 
- [ ] https://github.com/ipfs/go-ipfs/issues/1082
- [ ] https://github.com/ipfs/go-ipfs/issues/1195
- [ ] docker registry fun

#### @cryptix 
- [ ] https://github.com/ipfs/go-ipfs/issues/1214 (verify)
- [ ] https://github.com/ipfs/go-ipfs/issues/1050

#### @krl
- [ ] https://github.com/ipfs/webui/issues/39
- [ ] https://github.com/ipfs/webui/issues/40
- [ ] https://github.com/ipfs/webui/issues/43
- [ ] https://github.com/ipfs/webui/issues/48
- [ ] https://github.com/ipfs/webui/issues/50

#### @jbenet
- [x] https://github.com/jbenet/go-peerstream/issues/5 -- https://github.com/ipfs/go-ipfs/pull/1227
- [ ] Protocol wire spec
- [ ] swapping out the stream muxer (or having multiplex in go)
- [ ] DHT Records Spec!
- [ ] Keystore Spec!

#### @whyrusleeping
- [ ] bitswap perf https://github.com/ipfs/go-ipfs/issues/1218
  - waiting on merge
- [ ] better add perf https://github.com/ipfs/go-ipfs/issues/1223
- [ ] decimate eventlog https://github.com/ipfs/go-ipfs/issues/1224

#### @tv42 
- [ ] pinning in blockstore
- [ ] s3 datastore

#### triage
- [ ] https://github.com/ipfs/go-ipfs/issues/1211
- [ ] https://github.com/ipfs/go-ipfs/issues/1153
- [ ] https://github.com/ipfs/go-ipfs/issues/1197
- [ ] https://github.com/ipfs/go-ipfs/issues/1130
- [ ] https://github.com/ipfs/go-ipfs/issues/1216
- [ ] daemon init cleanup?
- [ ] stable resolution (tests)
- [ ] electron shell app (@krl or @jbenet) 
- [ ] symlinks (ran into this again) (@whyrusleeping) 
- [ ] write a migration for pinning (@whyrusleeping or @jbenet)

**Comment by whyrusleeping** [comment]
Everyone please let me know if the bugs under you are things you can get to this week or not. Also, if you would all take a look through bugs under 'triage' and claim those as you have time, that would be awesome.

**Comment by jbenet** [comment]
@krl @cryptix @wking -- could you check in with @whyrusleeping and me to go over things? Right now your things are tentative and want to figure them out with you. 

We'd like to also take care of the "traige" issues this week, not sure whether "assigning" or "pick up as we go" is a good idea.

**Comment by jbenet** [comment]
Also, i think we should endeavor to make sure that all the tasks are their own issue somewhere. that way we can have all the details specified there. I'll go through and make issues out of mine-- want to do the same for yours?

**Comment by krl** [comment]
Updated milestones for 0.2 release, will postpone all pinning stuff until that is settled upstream.

**Comment by jbenet** [comment]
- [x] pres at the W3C
- [x] https://github.com/jbenet/git-push-each
